api/src/contexts/users/controllers/UpdateUserProfileController.ts

```diff
@@ -0,0 +1,37 @@
+import express from 'express';
+import { IUpdateUserProfileInteractor } from '../usecases/IUpdateUserProfileInteractor';
+import { UpdateUserProfileInput } from '../interactors/UpdateUserProfileInteractor';
+
+export class UpdateUserProfileController {
+  constructor(
+    private readonly updateUserProfileInteractor: IUpdateUserProfileInteractor,
+  ) {}
+
+  async execute(req: express.Request, res: express.Response) {
+    try {
+      const userId = parseInt(req.params.id, 10);
```

Comment:
userIdはrequest contextのuserから取得できるので、そっちを使うようにしたいです！
```suggestion
      const userId = req.user.userId;
```